Studies/NeuralNetEstimator/Reco/create_config.py

```diff
@@ -0,0 +1,15 @@
+import io 
+import yaml
+
+cfg = {
```

Comment:
what is the purpose of this script? Why not create a `yaml` config by hand and commit it instead?